examples/js/datalayer.mocks.1.js

```diff
@@ -427,13 +419,12 @@ governing permissions and limitations under the License.
       // -----------------------------------------------------------------------------------------------------------------
 
       adobeDataLayer.push({
-        data: {
```

Comment:
this test does not make sense anymore as there is no invalid format: I'd remove it.